tests/subsys/bluetooth/controller/testcase.yaml

```
@@ -0,0 +1,8 @@
+tests:
+  bluetooth.controller:
+    platform_allow: native_sim
```

`platform_allow` section should be typed like list.
```suggestion
    platform_allow: 
        - native_sim
```